tensorflow_probability/python/optimizer/bfgs.py

```diff
@@ -81,7 +81,8 @@ def minimize(value_and_gradients_function,
              stopping_condition=None,
              validate_args=True,
              max_line_search_iterations=50,
-             name=None):
+             name=None,
+             line_search_kwargs={}):
```

Comment:
We have a pretty strong convention in TFP of letting the `name` argument be last.

Also please use `None` as a default value and explicitly set it to `{}` if `None` in the body, because `{}` is not safe as a default argument (see lint error on Travis).